bottle.py

```diff
@@ -69,10 +69,10 @@ def _cli_patch(cli_args):  # pragma: no coverage
 ###############################################################################
 
 
-import base64, cgi, email.utils, functools, hmac, imp, itertools, mimetypes,\
-        os, re, tempfile, threading, time, warnings, weakref, hashlib
```

Comment:
The most conservative change needed for this PR would probably be
```
import base64, cgi, email.utils, functools, hmac, itertools, mimetypes,\
        os, re, tempfile, threading, time, warnings, weakref, hashlib, \
        importlib.utils
```
depending on the linting configuration (line max width and whatnot)